### PR TITLE
Examples: SKLearn Pipelines and scheduling SKLearn

### PR DESCRIPTION
**Pull request by dgallitelli**
*Issue #, if available:*
N/A

*Description of changes:*
Two new notebooks: one for learning how to use SKLearn Pipelines in Amazon SageMaker, the other for scheduling SKLearn training and inference with SageMaker Pipelines.

*Testing done:*
No

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

- [x] I have read the [CONTRIBUTING](https://github.com/aws/amazon-sagemaker-examples/blob/master/CONTRIBUTING.md) doc and adhered to the example notebook best practices
- [x] I have updated any necessary documentation, including [READMEs](https://github.com/aws/amazon-sagemaker-examples/blob/master/README.md)
- [x] I have tested my notebook(s) and ensured it runs end-to-end
- [x] I have linted my notebook(s) and code using `tox -e black-format,black-nb-format`

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: sagemaker-pipelines/tabular/sklearn-scheduled-training-and-inference/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Scheduling SKLearn with Amazon SageMaker Pipelines
+
+In this notebook, we will use Amazon SageMaker Pipelines to create two workflows with Scikit-Learn. We will create a pipeline that preprocess data and trains a model (we will use scikit-learn Pipeline), then we will schedule inference with SageMaker Batch Transform.
```
Comment (jkroll-aws) [inline sagemaker-pipelines/tabular/sklearn-scheduled-training-and-inference/README.md:3]: Typo - change `preprocess` to `preprocesses`.

File: sagemaker-pipelines/tabular/sklearn-scheduled-training-and-inference/training.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+from shutil import copy
+
+# inference function - tells SageMaker how to load the model and do the prediction
+def model_fn(model_dir):
+    preproc = load(os.path.join(model_dir, "preproc.joblib"))
+    model = load(os.path.join(model_dir, "model.joblib"))
+    pipe = Pipeline([
+        ('preproc', preproc),('model', model)
+    ])
+    return pipe
```
Comment (jkroll-aws) [inline sagemaker-pipelines/tabular/sklearn-scheduled-training-and-inference/training.py:16]: Add newline between functions for readability.

File: sagemaker-pipelines/tabular/sklearn-scheduled-training-and-inference/scheduled-sm-pipelines-sklearn.ipynb [hunk 3]
```diff
@@ -0,0 +1,697 @@
+{
```
Comment (jkroll-aws) [inline sagemaker-pipelines/tabular/sklearn-scheduled-training-and-inference/scheduled-sm-pipelines-sklearn.ipynb:1]: <p>SageMaker Python SDK is installed in SageMaker notebooks and Studio by default.&nbsp;</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2755/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (jkroll-aws) [inline sagemaker-pipelines/tabular/sklearn-scheduled-training-and-inference/scheduled-sm-pipelines-sklearn.ipynb:1]: <p>Clear notebook outputs.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2755/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

**Comment by dgallitelli** [comment]
Updated the code with the above suggestions.

**Comment by zhaoqizqwang** [comment]
Closing this PR since it was created a long time ago and didn't pass all the checks. Feel free to open a new pull request instead.

**Pull request closed without merging**